mantle/platform/api/gcloud/compute.go

```diff
@@ -147,9 +147,9 @@ func (a *API) mkinstance(userdata, name string, keys []*agent.Key, opts platform
 		})
 	}
 	// create confidential instance
-	if a.options.Confidential {
+	if a.options.ConfidentialType == "SEV" || a.options.ConfidentialType == "SEV_SNP" {
 		instance.ConfidentialInstanceConfig = &compute.ConfidentialInstanceConfig{
-			EnableConfidentialCompute: true,
+			ConfidentialInstanceType: a.options.ConfidentialType,
```

Comment:
Minor: let's reorder the commits so that the vendor bump happens first so that this field is valid? That should silence golangci-lint.